### PR TITLE
ember-cli upgrade

### PR DESCRIPTION
**Pull request by aklkv**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,16 +25,13 @@ matrix:
 
 before_install:
   - npm config set spin false
-  - npm install -g bower
-  - bower --version
-  - npm install phantomjs-prebuilt
+  - npm install -g phantomjs-prebuilt
```
Comment (mansona) [inline .travis.yml]: If you upgraded this to a newer ember-cli this wouldn't be required 🎉

File: package.json [hunk 2]
```diff
@@ -2,54 +2,52 @@
   "name": "ember-cli-deploy-newrelic",
   "version": "0.0.0",
   "description": "The default blueprint for ember-cli addons.",
+  "keywords": [
+    "ember-addon",
+    "ember-cli-deploy-plugin"
+  ],
+  "license": "MIT",
+  "author": "",
```
Comment (mansona) [inline package.json]: You missing an author entry?

File: package.json [hunk 3]
```diff
@@ -2,54 +2,52 @@
   "name": "ember-cli-deploy-newrelic",
   "version": "0.0.0",
```
Comment (mansona) [inline package.json]: what version is it?

File: package.json [hunk 4]
```diff
@@ -2,54 +2,52 @@
...
+    "ember-addon",
+    "ember-cli-deploy-plugin"
+  ],
+  "license": "MIT",
+  "author": "",
   "directories": {
     "doc": "doc",
     "test": "tests"
   },
+  "repository": "",
```
Comment (mansona) [inline package.json]: should probably add

File: package.json [hunk 5]
```diff
@@ -2,54 +2,52 @@
...
-    "ember-load-initializers": "^0.5.1",
-    "ember-resolver": "^2.0.3",
-    "ember-welcome-page": "^1.0.1",
-    "loader.js": "^4.0.1"
+    "ember-disable-prototype-extensions": "^1.1.2",
+    "ember-export-application-global": "^2.0.0",
+    "ember-load-initializers": "^1.0.0",
+    "ember-resolver": "^4.2.1",
+    "ember-source": "^2.13.3",
+    "ember-welcome-page": "^3.1.1",
```
Comment (mansona) [inline package.json]: do you need welcome-page? is it being used by the dummy app?

File: tests/dummy/app/templates/application.hbs [hunk 6]
```diff
@@ -0,0 +1,5 @@
+{{!-- The following component displays Ember's default welcome message. --}}
+{{welcome-page}}
```
Comment (mansona) [inline tests/dummy/app/templates/application.hbs]: do you need this?

File: package.json (hunk 5 above)
Comment (aklkv) [reply package.json]: this comes with default blueprint, there is no harm in it being in devDeps

File: package.json (hunk 3 above)
Comment (aklkv) [reply package.json]: I have not released this one just yet

**Pull request merged**